Packs/CTIX/Integrations/CTIX/CTIX.py

```diff
@@ -0,0 +1,423 @@
+        else:
+            params = {"q": domain_string}
+
+        url_suffix = "objects/indicator/"
+        client_url = self.base_url + url_suffix
+        return self.http_request(full_url=client_url, **params)
+
+    def get_url_details(self, url: list, enhanced: bool = False):
+        """Gets the URL Details
+
+        :type url: ``list``
+        :param url: url name
+        :type enhanced: ``bool``
+        :param enhanced: Enhanced search flag
+
+        :return: dict containing the url details as returned from the API
+        :rtype: ``Dict[str, Any]``
+        """
+
+        url_string = ",".join(url)
+
+        if enhanced and argToBoolean(enhanced):
+            params = {"enhanced_search": url_string}
+        else:
+            params = {"q": url_string}
+
+        url_suffix = "objects/indicator/"
+        client_url = self.base_url + url_suffix
+        return self.http_request(full_url=client_url, **params)
+
+    def get_file_details(self, file: list, enhanced: bool = False):
+        """Gets the File Details
+
+        :type file: ``list``
+        :param file: file name
+
+        :type enhanced: ``bool``
+        :param enhanced: Enhanced search flag
+
+        :return: dict containing the file details as returned from the API
+        :rtype: ``Dict[str, Any]``
+        """
+
+        file_string = ",".join(file)
+        if enhanced and argToBoolean(enhanced):
+            params = {"enhanced_search": file_string}
+        else:
+            params = {"q": file_string}
+
+        url_suffix = "objects/indicator/"
+        client_url = self.base_url + url_suffix
+        return self.http_request(full_url=client_url, **params)
+
+
+''' HELPER FUNCTIONS '''
+
+
+def to_dbot_score(ctix_score: int) -> int:
+    """
+    Maps CTIX Score to DBotScore
+    """
+    if ctix_score == 0:
+        dbot_score = Common.DBotScore.NONE  # unknown
+    elif ctix_score <= 30:
+        dbot_score = Common.DBotScore.GOOD  # good
+    elif ctix_score <= 70:
+        dbot_score = Common.DBotScore.SUSPICIOUS  # suspicious
+    else:
+        dbot_score = Common.DBotScore.BAD
+    return dbot_score
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client):
+    """
+    Performs basic get request to get sample ip details.
+    """
+    client.test_auth()
+    # test was successful
+    demisto.results('ok')
+
+
+def ip_details_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    ip command: Returns IP details for a list of IPs
+    """
+    ip_addresses_string = args.get('ip')
+    ip_addresses_array = argToList(ip_addresses_string)
+    invalid_ips = []
+
+    for ip_address in ip_addresses_array:  # Check for Valid IP Inputs
+        if not is_ip_valid(ip_address, accept_v6_ips=True):
+            invalid_ips.append(ip_address)
+
+    if invalid_ips:
+        return_warning('The following IP Addresses were found invalid: {}'.format(', '.join(invalid_ips)))
```

Comment:
```suggestion
        return_warning('The following IP Addresses were found invalid: {}'.format(', '.join(invalid_ips)), exit=len(invalid_ips) == len(ip_addresses_array))
```
please add the same for all the relevant `return_warning` calls